contributing_to_docs/doc_guidelines.adoc

```diff
@@ -352,6 +352,25 @@ xref:../baz/zig.adoc#baz-zig[see the ZIG manual for more]
 You must use the .adoc extension in order for the link to work correctly and you must specify an anchor ID.
 ====
 
+== Embedding an external file
+
+You can embed content hosted outside the link:https://github.com/openshift/openshift-docs[openshift-docs]
+GitHub repository by using the `include` directive to target the URI of a raw
+file. This is helpful for cases where content frequently changes; you embed the raw
+file and the content auto-updates based on the changes made to the content on its
+host site.
+
+For example, if you want to embed the link:https://github.com/openshift/installer/blob/release-4.6/upi/azure/01_vnet.json[01_vnet.json] template, include the URI of its raw file version like this:
+
+----
+include::https://raw.githubusercontent.com/openshift/installer/release-4.6/upi/azure/01_vnet.json[]
+----
+
+[NOTE]
+====
+Embedding external files is restricted for files that change frequently, like templates. You must ensure that embedded files are QE verified before they are updated on their host site.
```

Comment:
@codyhoag - add a note to say that you must have an ack from Eng, QE and Docs to use the include link this way.